### PR TITLE
Raidboss: add ja/cn translation in e9s

### PR DESCRIPTION
**Pull request by MaikoTan**
https://github.com/quisquous/cactbot/blob/1b2bc6d65f02796782822beb87a8d1e2fbe6e2ff/ui/raidboss/data/05-shb/raid/e9s.txt#L13-L14

What is `(P/S)` by the way?

### CODE REVIEW

**Comment by panicstevenson** [comment]
> What is `(P/S)` by the way?

Protean / Stack: https://github.com/quisquous/cactbot/pull/2134#discussion_r540518760

**Comment by MaikoTan** [comment]
> > What is `(P/S)` by the way?
> 
> Protean / Stack: https://github.com/quisquous/cactbot/pull/2134#discussion_r540518760
> 
> 

Still misunderstanding, what is `Protean`?😂

**Comment by panicstevenson** [comment]
> Still misunderstanding, what is `Protean`?😂

"Clock Positions", but Protean is a reference to [Protean Wave](https://youtu.be/eBH371e9Jog?t=22) from A3S, which is where the mechanic was first introduced.

**Comment by r4ha** [comment]
I think those Japanese translate is incorrect. Let me review this.

**Comment by MaikoTan** [comment]
> I think those Japanese translate is incorrect. Let me review this.

Sure!

**Review by r4ha — CHANGES_REQUESTED** [review]
`苗木を捨て` feels like feels like there is a continuation. not an end of sentence.
so there needs to be `苗木を捨て`. others too.

File: ui/raidboss/data/05-shb/raid/e9s.js [hunk 1]
```diff
@@ -229,6 +232,8 @@ export default {
           en: 'Away From Tethered Walls',
           de: 'Weg von den verbundenen Wänden',
           fr: 'Éloignez-vous des murs liés',
+          ja: '線が繋がった外周に離れ',
```
[the remark this answers is not in the document]
Comment (MaikoTan) [reply ui/raidboss/data/05-shb/raid/e9s.js:235]: This is the ["活性弾" gimmick (2:23)](https://youtu.be/_fdYxEVmCMs) I think.

ボスと外周に紫色の線(横あるいは縦)が繋がれる、繋がれた方向から巨大な円範囲攻撃が発生です。

[the remark this answers is not in the document]
Comment (r4ha) [reply ui/raidboss/data/05-shb/raid/e9s.js:235]: ah I see. then `ja: '線が繋がった方へ',` is good.

[the remark this answers is not in the document]
Comment (MaikoTan) [reply ui/raidboss/data/05-shb/raid/e9s.js:235]: "繋がれなかった方へ" is correct I think.

[the remark this answers is not in the document]
Comment (r4ha) [reply ui/raidboss/data/05-shb/raid/e9s.js:235]: yes yes im so dumb sorry

**Pull request merged**